src/coreclr/src/gc/sample/GCSample.vcxproj

```diff
@@ -17,13 +17,13 @@
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
+  <PropertyGroup Condition="'$(ConfigurationGroup)|$(ArchGroup)'=='Debug|Win32'" Label="Configuration">
```

Comment:
We could probably create a property for this value and use that for the condition. Also, spaces in between `'` and `==`.

Reply:
I don't think it's worth it. I just replaced the property name here.